src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -1301,6 +1305,43 @@ def set_up_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
         mc = super().set_up_network_profile(mc)
         network_profile = mc.network_profile
 
+        ip_families = self.context.raw_param.get('ip_families')
+
+        if ip_families:
+            network_profile.ip_families = ip_families.split(',')
+
+        pod_cidrs = self.context.raw_param.get('pod_cidrs')
+        if pod_cidrs:
+            network_profile.pod_cidrs = pod_cidrs.split(',')
+
+        service_cidrs = self.context.raw_param.get('service_cidrs')
+        if service_cidrs:
+            network_profile.service_cidrs = service_cidrs.split(',')
+
+        ipv6_count = self.context.raw_param.get('load_balancer_managed_outbound_ipv6_count')
+        if ipv6_count:
+            print('pre + ' + str(network_profile))
```

Comment:
Please remove `print` or replace it with `logger.debug` before merging this PR.

Reply:
removed